### PR TITLE
[ADD] FileBot 4.8.5

### PR DESCRIPTION
**Pull request by rednoah**
* moved to `extras` from https://github.com/ScoopInstaller/Main/pull/370

### CODE REVIEW

File: bucket/filebot.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "https://www.filebot.net/",
+    "description": "The ultimate TV and Movie Renamer",
```
Comment (Ash258) [inline bucket/filebot.json]:
```suggestion
    "description": "Movies, TV Shows and Anime files organizer and renamer.",
```

Comment (rednoah) [reply bucket/filebot.json]: I'd keep the official branding, but if you prefer the one you write, then that's perfectly fine too.

Comment (Ash258) [reply bucket/filebot.json]: `The ultimate` is marketing stuff, which has anything to do with application description.

Comment (rednoah) [reply bucket/filebot.json]: I auto-generate manifests for brew / chocolatey / scoop / etc in my build:
```
package.synopsis: The ultimate TV and Movie Renamer
package.description: FileBot is the ultimate tool for organizing and renaming your Movies, TV Shows and Anime as well as fetching subtitles and artwork. It's smart and just works.
```
No worries. I'll just hard code description you like in my scoop template.

Comment (Ash258) [reply bucket/filebot.json]: You do not need to generate scoop manfiest in each build. All manifests are automatically updated on hour basis.  (https://github.com/ScoopInstaller/Excavator)

Comment (Ash258) [reply bucket/filebot.json]: As soon as PR is merged and new version is relased and identified, manifest will be updated in repository.

Comment (rednoah) [reply bucket/filebot.json]: Alright. That makes sense. Thanks!

Comment (Ash258) [reply bucket/filebot.json]: Your welcome.

**Comment by wenmin92** [comment]
@Ash258 This is a paid APP, that's OK?

**Comment by rednoah** [comment]
FileBot is indeed commercial proprietary software. I've created a manifest here because one of my customers asked me to support `scoop install filebot` if possible.

**Pull request merged**